mongodb/Package.md

```diff
@@ -1,155 +1,21 @@
-# Ballerina MongoDB Connector
-
 Connects to MongoDB from ballerina 
 
-## Module Overview
-
-The Mongo DB connector allows you to connect to a Mongo DB from Ballerina and perform various operations such as `getDatabaseNames`, `getCollectionNames`, `count`, `listIndices`, `find`, `insert`, `update`, and `delete`.
-
-## Prerequisites
+## Package overview
 
-* A mongodb with username and password
+The `ballerinax/mongodb` is a [Ballerina](https://ballerina.io/) connector for MongoDB.
```

Comment:
Shall we use the connector name. Please check other connectors and follow a similar approach.

Reply:
Here Connector name, Package name and module name all are same `MongoDB`.

Reply:
Since this is the format across all the connector, Let's keep it, as it is for now ,as we discussed in offline.